microsoft-edge/webview2/get-started/winui.md

```diff
@@ -246,7 +233,7 @@ You may use host apps to inject JavaScript code into WebView2 controls at runtim
 As an example, add scripts that send an alert when a user navigates to non-HTTPS sites.  Modify the `EnsureHttps` function to inject a script into the web content that uses [ExecuteScriptAsync][Webviews2ReferenceWpfMicrosoftWebExecutescriptasync].  
```

Comment:
Also, in the second bullet under the "Scripting" heading, the second bullet: "Run it before any other script included in the HTML document is run." reads better as "Run it before any other script included in the HTML document runs."

Reply:
changed